src/bloom/utils.rs

```diff
@@ -616,14 +618,23 @@ mod tests {
                     fp_count += 1;
                 }
                 Ok(1) => {
+                    if let Some(BloomError::NonScalingFilterFull) = expected_error {
+                        panic!("A non scaling filter that is full shouldn't be able to have new items added successfully");
+                    }
```

Comment:
If something like this is possible, it will be more generic: 

```suggestion
                    if let Some(err) = expected_error {
                        panic!("Expected error on the bloom object during during item add: {:?}", err);
                    }
```